docs/speech-to-text/realtime/realtime_diarization.mdx

```diff
@@ -174,18 +174,19 @@ Transcripts are returned independently for each channel, with the `channel` prop
 }
 ```
 
-Note that the `channel` property will be returned for `AddTranscript` and `AddPartialTranscript` messages only. Features such as [audio events](/speech-to-text/features/audio-events) do not currently include this property. To request this feature, please contact [support](https://support.speechmatics.com).
+:::warning
+The `channel` property will be returned for `AddTranscript` and `AddPartialTranscript` messages only. 
+Features such as [audio events](/speech-to-text/features/audio-events), [translation](/speech-to-text/features/translation) and [end of turn detection](/speech-to-text/features/end-of-turn) do not currently include this property. To request this feature, please contact [support](https://support.speechmatics.com).
```

Comment:
`/speech-to-text/features/end-of-turn` is a broken link according to CI, should be: `/speech-to-text/realtime/end-of-turn`